PixiEditor/ViewModels/SubViewModels/Main/WindowViewModel.cs

```diff
@@ -1,15 +1,13 @@
 ﻿using AvalonDock.Layout;
-using PixiEditor.Helpers;
+using GalaSoft.MvvmLight.CommandWpf;
```

Comment:
This is fine for now, but we will get rid of this after merging #306

Reply:
alright